api/src/common/blocks/link-list.block.ts

```diff
@@ -0,0 +1,4 @@
+import { createListBlock } from "@comet/blocks-api";
+import { TextLinkBlock } from "@src/common/blocks/text-link.block";
```

Comment:
```suggestion
import { TextLinkBlock } from "./text-link.block";
```
(you need to run eslint)

Reply:
I think it is correct like it is 🤔

Reply:
No, that should be relative, as both blocks are in `src/common/blocks`

Reply:
https://github.com/vivid-planet/comet-starter/pull/282/commits/81283fb66252f7303c3c146d944050bfbfa7505b